src/widgets/courses-school/ui/courses.tsx

```diff
@@ -16,7 +18,11 @@ export const Courses = () => {
   const size = useWindowSize();
   const laptopScreenBreakPoint = 1440;
   const tabletScreenBreakPoint = 810;
-  const coursesData = selectCourses(courses);
+  const coursesData = getActualDataList({
+    dataList: courses,
+    actualDelayInDays: COURSE_STALE_AFTER_DAYS,
+    filtered: true,
+  }) as Course[];
```

Comment:
Let's use `isCourse` type guard here

Reply:
I'll use it in helper so it will return specified type and no assertions will be in code. But but will make it in issue #400 as @ansivgit asked to separate to small tasks.